src/framework/mlt_factory.c

```diff
@@ -137,10 +144,14 @@ mlt_repository mlt_factory_init( const char *directory )
 		#endif
 		mlt_properties_set( global_properties, "MLT_APPDIR", appdir );
 		free( appdir );
-#elif defined(__APPLE__)  && defined(RELOCATABLE)
+#elif defined(RELOCATABLE)
 		char path[1024];
 		uint32_t size = sizeof( path );
+#  ifdef __APPLE__
 		_NSGetExecutablePath( path, &size );
+#  else
+                readlink( "/proc/self/exe", path, size );
```

Comment:
Incorrect white space to indent - should be tabs.

Reply:
right, fixed!